### ISSUE TITLE
Reports to downgrade flutter

### ISSUE THREAD

**Issue by danielgomezrico** [pszklarska/FlutterPubVersionChecker#22]
Version: 1.2.5
Ide: Intellij Communiity 2020.2.2

![image](https://user-images.githubusercontent.com/76348/94467502-b17ef680-0188-11eb-9164-0fdc18626f7a.png)

**Comment by danielgomezrico** [comment]
@pszklarska Im getting an error:

Plugin version: 1.2.5
Intellij version: 2021.2.2
Flutter:2.5.1

```kotlin
kotlin.KotlinNullPointerException
	at pl.pszklarska.pubversionchecker.DependencyChecker.getLatestVersion(DependencyChecker.kt:28)
	at pl.pszklarska.pubversionchecker.FileParser.mapToVersionDescription(FileParser.kt:56)
	at pl.pszklarska.pubversionchecker.FileParser.access$mapToVersionDescription(FileParser.kt:12)
	at pl.pszklarska.pubversionchecker.FileParser$getVersionsFromFile$$inlined$map$lambda$1.invokeSuspend(FileParser.kt:37)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:56)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:571)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:738)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:678)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:665)
```

**Comment by pszklarska** [comment]
Hey @danielgomezrico, I fixed your issue in this PR: https://github.com/pszklarska/FlutterPubVersionChecker/pull/41. It will be included in the newest release (1.3.1) which should be ready to download from the Market in the next few days :)

**Comment by danielgomezrico** [comment]
@pszklarska it does work now thanks :)
